K2Bridge/Visitors/HistogramVisitor.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// See LICENSE file in the project root for full license information.
+
+namespace K2Bridge.Visitors
+{
+    using System;
+    using K2Bridge.Models.Request.Aggregations;
+    using K2Bridge.Models.Response;
+    using K2Bridge.Utils;
+
+    /// <content>
+    /// A visitor for the <see cref="HistogramAggregation"/> element.
+    /// </content>
+    internal partial class ElasticSearchDSLVisitor : IVisitor
+    {
+        /// <inheritdoc/>
```

Comment:
can we add in the comment how the generated kql looks like, if multiple options that list all. so readers can easily understand this file (I also think this should be done for all other visitors)

Reply:
Done